hoot-services/src/main/java/hoot/services/controllers/osm/user/UserResource.java

```diff
@@ -111,6 +105,27 @@ public Response get(@Context HttpServletRequest request, @PathParam("userId") Lo
         }
     }
 
+    /**
+     * Service method endpoint for retrieving Hoot user
+     *
+     * GET hoot-services/osm/api/0.6/user/details
+     *
+     * @return Response with the user's information
+     */
+    @GET
+    @Path("/details")
+    public Response getDetails(@Context HttpServletRequest request) throws ParserConfigurationException {
+        Users user = Users.fromRequest(request);
+        String contentType = null;
```

Comment:
Do we want to create a helper function that sets up the response after a user is retrieved that these 3 endpoints can call since they seem to be using the same code?
- `/{userId}`
- `/details`
- `/name/{displayName}`